packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -383,30 +383,48 @@ Resources:
       Policies:
         - PolicyDocument:
             Statement:
-              - Action:
-                  # Permissions needed in CodePipeline.
-                  # S3 and KMS are needed on '*',
-                  # as we don't know the IDs of the CodePipeline's bucket and key in advance
+              - Sid: CloudFormationPermissions
+                Effect: Allow
+                Action:
                   - cloudformation:CreateChangeSet
```

Comment:
Can you please add CreateStack and UpdateStack?

Reply:
@eladb the `CreateChangeSet` allows creating an updating stacks by specifying `ChangeSetType ` as `Update`. See docs: https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_CreateChangeSet.html

This should be sufficient, unless Im missing some use case

Reply:
In some scenarios (eg the GH action I've been using to implement the CDK pipelines backend for GitHub) we may _need_ to use CreateStack and UpdateStack